openwisp_controller/config/models.py

```diff
@@ -146,29 +151,6 @@ class Meta(AbstractTaggedTemplate.Meta):
         abstract = False
 
 
-class Template(ShareableOrgMixin, AbstractTemplate):
-    """
-    openwisp-controller Template model
-    """
-    tags = TaggableManager(through='config.TaggedTemplate', blank=True,
-                           help_text=_('A comma-separated list of template tags, may be used '
-                                       'to ease auto configuration with specific settings (eg: '
-                                       '4G, mesh, WDS, VPN, ecc.)'))
-    vpn = models.ForeignKey('config.Vpn',
-                            verbose_name=_('VPN'),
-                            blank=True,
-                            null=True,
-                            on_delete=models.CASCADE)
-
-    class Meta(AbstractTemplate.Meta):
-        abstract = False
-        unique_together = (('organization', 'name'), )
-
-    def clean(self):
-        self._validate_org_relation('vpn')
-        super(Template, self).clean()
-
-
```

Comment:
why are you moving the `Template` model down? I would be very grateful if you avoid to do this kind of changes because it pollutes the PR and adds more work on the reviewer (me) to understand what is going on

Reply:
what about this comment? Can you follow this instruction please?